code/modules/clothing/clothing.dm

```diff
@@ -477,6 +477,13 @@ BLIND     // can't see anything
 			. += mutable_appearance('icons/effects/item_damage.dmi', "damaged[blood_overlay_type]")
 		if(blood_DNA)
 			. += mutable_appearance('icons/effects/blood.dmi', "[blood_overlay_type]blood")
+		var/mob/living/carbon/human/M = loc

```

Comment:
If you want to make a PR to this branch, then I'd certainly merge it, since it'd probably be better than what I managed to do(which is break it).